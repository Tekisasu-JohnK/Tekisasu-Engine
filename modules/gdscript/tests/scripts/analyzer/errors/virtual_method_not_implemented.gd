func test():
	_get_property_list()
