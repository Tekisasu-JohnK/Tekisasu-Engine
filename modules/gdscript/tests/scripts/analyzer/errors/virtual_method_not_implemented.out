GDTEST_ANALYZER_ERROR
Cannot call virtual function "_get_property_list()" because it hasn't been defined.
